### PR TITLE
Enhance cluster support

### PR DESCRIPTION
**Pull request by cpg1111**

### CODE REVIEW

File: pylxd/models/tests/test_cluster_member.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ def test_get(self):
         member = self.client.cluster.members.get("an-member")
 
         self.assertEqual("https://10.1.1.101:8443", member.url)
+        self.assertEqual("x86_64", member.architecture)
+        self.assertEqual([], member.roles)
```
Comment (albertodonato) [inline pylxd/models/tests/test_cluster_member.py:27]: mind converting asserts in this test to just plain `assert`s?
We've transitioned to pytest recently, would be good to gradually move everything to `assert`

**Review by albertodonato — APPROVED** [review]
+1 with a nit

**Comment by albertodonato** [comment]
Also you'll need a signoff line in each commit (to pass the DCO test)

File: pylxd/models/tests/test_cluster_member.py (hunk 1 above)
Comment (cpg1111) [reply pylxd/models/tests/test_cluster_member.py:27]: to confirm, you mean `assert "x86_64 == member.architecture`, etc?

Comment (albertodonato) [reply pylxd/models/tests/test_cluster_member.py]: yes

Comment (albertodonato) [reply pylxd/models/tests/test_cluster_member.py]: although, `assert member.architecture == "x86_64"` seems more natural to me

**Pull request merged**